### DISCUSSION TITLE
Please verify that the package.json has a valid "main" entry  For rxjs

### DISCUSSION THREAD

**Discussion by MaleWeb** [outline/outline#2834] [Self Hosting]
When I was installing, I reported errors frequently。I tried the following scheme。
1.Delete in project file  package-lock.json  
2.npm install
3.restart
Unfortunately, this problem still cannot be solved

![image](https://user-images.githubusercontent.com/20508968/145194723-1b6534a0-f15e-400e-8c5f-d02a8b5355e0.png)

**Comment by tommoor** [comment]
How are you running/installing? `concurrently` is only used in the development environment.
